permissionless-batches/docker-compose.yml

```
@@ -0,0 +1,121 @@
+      db:
+        condition: service_healthy
+
+
+  proving-service-chunk:
+    image: scrolltech/sdk-cloud-prover:sindri-v0.0.5
+    platform: linux/amd64
+    command: "--config /app/config.json"
+    profiles:
+      - proving
+    environment:
+          PROVER_NAME_PREFIX: "sindri_chunk"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 10

(indentation)